src/components/ebay-toggle-button-group/index.marko

```diff
@@ -2,25 +2,24 @@ import { processHtmlAttributes } from "../../common/html-attributes";
 
 $ const {
     class: inputClass,
-    columns,
     layoutType: baseLayoutType,
     buttons = [],
     ...htmlInput
 } = input;
 
-<ul
+<div
```

Comment:
I think this is missing the breakpoint-specific attributes.

Also, the `a11y` attributes need to stay on the `<ul>`.